src/Samples/WsHttpBindings/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Diagnostics;
+using System.Net;
+using CoreWCF.Configuration;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+
+namespace NetCoreServer
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            IWebHost host = CreateWebHostBuilder(args).Build();
```

Comment:
on NetCoreServer we could use the netcoreapp3.1/net5.0 variation which uses the generic host.

Reply:
Can you point me to what you mean?

Reply:
sure. please see below
```c#
public class Program
{
    public static void Main(string[] args) => CreateHostBuilder(args).Build().Run();
    
    public static IHostBuilder CreateHostBuilder(string[] args) =>
        Host.CreateDefaultBuilder(args)
            .ConfigureWebHostDefaults(webBuilder =>
            {
                webBuilder.UseKestrel(options =>
                {
                    options.AllowSynchronousIO = true;
                });
                webBuilder.UseStartup<Startup>();
            });
}
```